### PR TITLE
chore(components): make prettier ignore the CHANGELOG.md since it's autogenerated

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #

### PR Checklist

- ~~[ ] All necessary documentation has been adapted.~~
- ~~[ ] The implemented feature is covered by an appropriate test.~~

### CODE REVIEW

File: components/.prettierrc.js
Comment (fengelniederhammer) [inline components/.prettierrc.js:1]: I like JS more than plain JSON files, thus this change.

File: components/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/** @type {import("prettier").Config} */
```
Comment (JonasKellerer) [inline components/.prettierrc.js:1]: This cant be a ts file?

**Review by JonasKellerer — APPROVED** [review]
LGTM

File: components/.prettierrc.js (hunk 1 above)
Comment (fengelniederhammer) [reply components/.prettierrc.js:1]: https://prettier.io/docs/en/configuration doesn't list TS, unfortunately.

**Pull request merged**